src/ScrollTo.jsx

```diff
@@ -37,10 +37,21 @@ class ScrollTo extends Component {
         container.scrollTop = y;
       });
     }
-  };
+  }
+
+  scrollById(id, x, y) {
+    var node = document.getElementById(id);
+    console.log("HI");
```

Comment:
@takeyan1004 Let's clean out the `console.log` statements here. We forgot to take them out.